packages/google_maps_flutter/google_maps_flutter_android/android/src/test/java/io/flutter/plugins/googlemaps/ConvertTest.java

```diff
@@ -49,10 +48,8 @@
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 import org.robolectric.RobolectricTestRunner;
-import org.robolectric.annotation.Config;
 
 @RunWith(RobolectricTestRunner.class)
-@Config(minSdk = Build.VERSION_CODES.LOLLIPOP)
```

Comment:
I think the @config annotation should stay and you should use the new minimum. 

minSdk= is supposed to run the tests for every version of android after that version.

Reply:
Maybe I misunderstood `@Config`. Does it restrict the APIs that this test *can* be run under, or does it *cause* the test to be run with multiple API levels? I thought it was the former.

If it's the latter, I'm skeptical we want this. We don't do this pretty much anywhere else in the repo, and I can't think of any reason we would want to multi-run things for the maps SDK in particular, since the SDK isn't system-API-level based.

Reply:
My understanding is it is the latter. Here is some documentation that I think supports that conclusion. https://robolectric.org/configuring/#configure-sdk-level. If you wanted to run on a specific set then you would use sdk = <SOME SET OF API LEVELS>. 

Looking through ConvertTest I agree but there are other places in packages where it does make sense to run with a minSdk or other broader android versions set. Primarily to make sure branching logic based on android api gets exercised. 

Update: a spot check of all of these google maps test entries shows minimal value in multi api level tests because they do not branch.

Reply:
I'll go ahead and remove them then; the fact that we only do this for these tests, and these tests don't seem like they benefit, makes me think that adding it was a result of a misunderstanding like mine.